From linux-usb-devel-admin@lists.sourceforge.net Sun Apr 16 19:19:07 2006
Message-ID: <4442FAFC.3070802@ipom.com>
From: Phil Dibowitz <phil@ipom.com>
To: Kenneth Crudup <kenny@panix.com>
Cc: linux-usb-devel@lists.sourceforge.net
Subject: USB: Storage: unusual devs update
Date: Sun, 16 Apr 2006 19:18:36 -0700

This patch removes the Protocol portion of the Iomega Click! device as it's not
needed. Not-needed message reported by Kenneth Crudup <kenny@panix.com>

Signed-off-by: Phil Dibowitz <phil@ipom.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---


---
 drivers/usb/storage/unusual_devs.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/storage/unusual_devs.h
+++ gregkh-2.6/drivers/usb/storage/unusual_devs.h
@@ -411,7 +411,7 @@ UNUSUAL_DEV(  0x050d, 0x0115, 0x0133, 0x
 UNUSUAL_DEV(  0x0525, 0xa140, 0x0100, 0x0100,
 		"Iomega",
 		"USB Clik! 40",
-		US_SC_8070, US_PR_BULK, NULL,
+		US_SC_8070, US_PR_DEVICE, NULL,
 		US_FL_FIX_INQUIRY ),
 
 /* Yakumo Mega Image 37
